systemtests/src/test/java/io/enmasse/systemtest/shared/standard/api/UserApiTest.java

```diff
@@ -4,6 +4,7 @@
  */
 package io.enmasse.systemtest.shared.standard.api;
 
+import static io.enmasse.systemtest.TestTag.ACCEPTANCE;
```

Comment:
Why do you add imports there? I don't see any usage of acceptance profile in slass

Reply:
I'd failed to rebase properly before I started.  That commit was already merged.